server/src/core/google-client.ts

```diff
@@ -0,0 +1 @@
+export const GOOGLE_CLIENT_ID = '868422357333-sbrtb81b6rdjbo9ag3qfdnqc2pnm6dct.apps.googleusercontent.com';
```

Comment:
Put this value in the `/server/env` file

Reply:
Oh yeah I didn't know where to put it. Thanks!